extensions/git/src/commands.ts

```diff
@@ -1370,9 +1370,10 @@ export class CommandCenter {
 				value = (await repository.getCommit(repository.HEAD.commit)).message;
 			}
 
+			const branchName = repository.getBranchName();
```

Comment:
If this returns `undefined`, won't we see `'undefined'` in the input box?

Reply:
No, it would be localized `commit message`. (There's ternary operator below)

Reply:
Ok, changed ternary `?` to `if` condition.